provisioning_templates/provision/preseed_default.erb

```diff
@@ -8,9 +8,16 @@ oses:
 %>
 <%
   proxy_string = host_param('http-proxy') ? " http://#{host_param('http-proxy')}:#{host_param('http-proxy-port')}" : ''
+  ansible_enabled = @host.respond_to?(:all_ansible_roles)
   salt_enabled = host_param('salt_master') ? true : false
   os_major = @host.operatingsystem.major.to_i
   squeeze_or_older = (@host.operatingsystem.name == 'Debian' && os_major <= 6)
+
+  additional_packages = ['lsb-release']
+  additional_packages << host_param('additional-packages')
+  additional_packages << 'python' if ansible_enabled
+  additional_packages << 'salt-minion' if salt_enabled
+  additional_packages = additional_packages.join(" ").split(" ").uniq().join(" ")
```

Comment:
Isn't `.split()` safer so you split on any whitespace?

Reply:
I you say so...